vrc-get-vpm/src/environment/unity_management.rs

```diff
@@ -211,8 +211,15 @@ fn default_unity_hub_path() -> &'static [&'static str] {
                 let home = std::env::var("HOME").expect("HOME not set");
                 format!("{}/Applications/Unity Hub.AppImage", home)
             };
-            static ref INSTALLATIONS: [&'static str; 3] =
-                [&USER_INSTALLATION, "/usr/bin/unity-hub", "/opt/unityhub/unityhub"];
+            static ref INSTALLATIONS: [&'static str; 4] =
+                [
+                    &USER_INSTALLATION,
+                    "/usr/bin/unity-hub",
+                    // apt package
+                    "/opt/unityhub/unityhub",
+                    // flatpak
+                    "/var/lib/flatpak/export/bin/com.unity.UnityHub",
```

Comment:
I made one easy mistake in my last comment referring to the path of unityhub, the word `export` is suppose to be plural for system wide installed flatpaks, so the correct path is `exports`.
`/var/lib/flatpak/exports/bin/com.unity.UnityHub`